01_temperature_processing.R

```diff
@@ -44,7 +45,7 @@ library(plyr)
 
 #Set the data file path and file that you want to process
 src_file <- '//deqlead02/Vol_Data/salmon-drift/2015/Fake2SDCWC2015CDO.xls'
-SubID <- 'XXXX' # Enter the submission ID from VolWQDB
+SubID <- '0020' # Enter the submission ID from VolWQDB
```

Comment:
Ok. I think I need to see the workflow written out to get a complete grasp on this. I think it's probably fine the way it is here for now.